app/admin/pages.rb

```diff
@@ -35,6 +35,10 @@
                     :cta_text,
                     :button_text,
                     :card,
+                    :display_successful_adoptions,
+                    :display_in_progress_adoptions,
+                    :display_unsuccessful_adoptions,
+                    :short_name,
```

Comment:
Not sure I follow.  short_name is a field in the map component.  Are you saying just make it a local variable on the pages.rb page?  How would it get set?

Reply:
It displays in the infowindow - 
![image](https://user-images.githubusercontent.com/60527788/194431202-53bbb0a7-1501-4004-bab2-b2164be127ee.png)

Reply:
Then should we rename the field to `map_info_window_text`? When I think of short name, I think of the short name of a practice, or in this case, the Page title. If it can be anything, I think it would make more sense to reflect the ambiguity of it. Just my two cents.

Reply:
Renamed the column to map_info_window_text.